snmp/datadog_checks/snmp/data/profiles/generic-router.yaml

```diff
@@ -1,12 +1,13 @@
-# Generic profile for routers
+# Generic profile for network devices
+# Note: This profile is deprecated, use generic-network instead
 #
 
 extends:
   - _base.yaml
-  - _generic-router-if.yaml
-  - _generic-router-ip.yaml
-  - _generic-router-tcp.yaml
-  - _generic-router-udp.yaml
-  - _generic-router-ospf.yaml
+  - _generic-if.yaml
+  - _generic-ip.yaml
+  - _generic-tcp.yaml
+  - _generic-udp.yaml
+  - _generic-ospf.yaml
 
 sysobjectid: 1.3.6.1.4.*
```

Comment:
@therve, what happens if there are two profiles with the same sysobjID?
(Renaming this was a nice to have so I'll undo it if there are issues)

Reply:
Oh, I see it raises a ConfigurationError. I'm going to leave this as is @florimondmanca